### PR TITLE
forward domain added & skin added

### PR DESCRIPTION
**Pull request by dhavalalite**
#### Hi! What is your PR about?

- [x] Add or edit a skin

- [x] Add a custom domain for your space

- [ ] Add an alias to migrate your space

### CODE REVIEW

**Review by Yara1990 — APPROVED** [review]
Looks good, merging now

**Review by Yara1990 — COMMENTED** [review]
Good to go live

**Comment by Yara1990** [comment]
@gokaiorg Kindly review the above fixes.

**Comment by Yara1990** [comment]
> 

@gokaiorg Kindly review the above fixes.

**Comment by gokaiorg** [comment]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?
To prevent conflicts, it is recommended to add your subdomain between two existing domains rather than at the end or beginning of the list.

**Comment by Yara1990** [comment]
@gokaiorg We have resolved it. Please approve the same.

File: skins/forward.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.forwardprotocol {
```
Comment (gokaiorg) [inline skins/forward.scss:1]: Your class name should be similar to file name.
```suggestion
.forward {
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**